src/components/RocketsPage.js

```diff
@@ -1,9 +1,24 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
+import RenderRocket from './RenderRocket';
+import '../styles/RenderRocket.scss';
 
-const RocketsPage = () => (
-  <section>
-    <h2> Rockets Page </h2>
-  </section>
-);
+function RocketsPage() {
+  const rocketsColl = useSelector((state) => state.rocketsReducer);
+  return (
+    <section>
+      <ul>
+        {rocketsColl.map((rocket) => (
+          <RenderRocket
+            key={rocket.id}
+            name={rocket.rocket_name}
+            desc={rocket.description}
+            img={rocket.flickr_images}
+          />
+        ))}
```

Comment:
Great job here. I did the same.
I think it's good to create a separate component whenever we have to map a list of data. We are on the same page. 🎉